airbyte-scheduler/persistence/src/main/resources/airbyte_jobs_table.sql

```diff
@@ -0,0 +1,78 @@
+-- types
```

Comment:
this seems to be the previous job table sql. should this be deleted?

Reply:
don't see any usages of this file

Reply:
You are right. This file should be removed.